09:28:10.6556-05:00 - ERROR -  - ------------- BEGIN Flow.Launcher.Plugin.Program exception -------------
09:28:10.6556-05:00 - ERROR -  - 
Exception full name: System.IO.FileNotFoundException
Error status: UNKNOWN
Class name: Win32
Calling method: ExeProgram
Program path: C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.5.0_x64__8wekyb3d8bbwe\msrdc.exe
InnerException number: 1
Exception message: C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.5.0_x64__8wekyb3d8bbwe\msrdc.exe
Exception error type: HResult -2147024894
Exception thrown in called method: System.Diagnostics.FileVersionInfo GetVersionInfo(System.String)
Possible interpretation of the error: File not found when trying to load the program from C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.5.0_x64__8wekyb3d8bbwe\msrdc.exe
Possible resolution: Not yet known
09:28:10.6556-05:00 - ERROR -  - ------------- END Flow.Launcher.Plugin.Program exception -------------
09:28:10.6840-05:00 - ERROR -  - ------------- BEGIN Flow.Launcher.Plugin.Program exception -------------
09:28:10.6840-05:00 - ERROR -  - 
Exception full name: System.IO.FileNotFoundException
Error status: UNKNOWN
Class name: Win32
Calling method: ExeProgram
Program path: C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.5.0_x64__8wekyb3d8bbwe\wslg.exe
InnerException number: 1
Exception message: C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.5.0_x64__8wekyb3d8bbwe\wslg.exe
Exception error type: HResult -2147024894
Exception thrown in called method: System.Diagnostics.FileVersionInfo GetVersionInfo(System.String)
Possible interpretation of the error: File not found when trying to load the program from C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.5.0_x64__8wekyb3d8bbwe\wslg.exe
Possible resolution: Not yet known
09:28:10.6840-05:00 - ERROR -  - ------------- END Flow.Launcher.Plugin.Program exception -------------
09:28:12.7314-05:00 - ERROR -  - ------------- BEGIN Flow.Launcher.Plugin.Program exception -------------
09:28:12.7314-05:00 - ERROR -  - 
Exception full name: System.Runtime.InteropServices.COMException
Error status: UNKNOWN
Class name: IShellLinkW
Calling method: retrieveTargetPath
Program path: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Accessories\Windows Media Player Legacy.lnk
InnerException number: 1
Exception message: Error HRESULT E_FAIL has been returned from a call to a COM component.
Exception error type: HResult -2147467259
Exception thrown in called method: Void GetDescription(System.Text.StringBuilder, Int32)
Possible interpretation of the error: Error caused likely due to trying to get the description of the program
Possible resolution: Not yet known
09:28:12.7314-05:00 - ERROR -  - ------------- END Flow.Launcher.Plugin.Program exception -------------
09:28:12.7908-05:00 - ERROR -  - ------------- BEGIN Flow.Launcher.Plugin.Program exception -------------
09:28:12.7908-05:00 - ERROR -  - 
Exception full name: System.Runtime.InteropServices.COMException
Error status: UNKNOWN
Class name: IShellLinkW
Calling method: retrieveTargetPath
Program path: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Administrative Tools\ODBC Data Sources (32-bit).lnk
InnerException number: 1
Exception message: Error HRESULT E_FAIL has been returned from a call to a COM component.
Exception error type: HResult -2147467259
Exception thrown in called method: Void GetDescription(System.Text.StringBuilder, Int32)
Possible interpretation of the error: Error caused likely due to trying to get the description of the program
Possible resolution: Not yet known
09:28:12.7908-05:00 - ERROR -  - ------------- END Flow.Launcher.Plugin.Program exception -------------
13:25:41.3310-05:00 - INFO  - Updater.UpdateApp - Future Release <{
  "SHA1": "AE2E89611857AD4D6FA76D59827AE2E1DD8BC4F9",
  "BaseUrl": null,
  "Filename": "FlowLauncher-1.17.2-full.nupkg",
  "Query": null,
  "Filesize": 105388595,
  "IsDelta": false,
  "StagingPercentage": null,
  "EntryAsString": "AE2E89611857AD4D6FA76D59827AE2E1DD8BC4F9 FlowLauncher-1.17.2-full.nupkg 105388595",
  "Version": {
    "Version": "1.17.2.0",
    "SpecialVersion": ""
  },
  "PackageName": "FlowLauncher"
}>
18:25:41.2461-05:00 - ERROR - Updater.UpdateApp - Check your connection and proxy settings to github-cloud.s3.amazonaws.com.
EXCEPTION OCCURS: System.Net.Http.HttpRequestException: No such host is known. (api.github.com:443)
 ---> System.Net.Sockets.SocketException (11001): No such host is known.
   at void System.Net.Sockets.Socket+AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at async bool System.Net.Sockets.Socket.ConnectAsync(SocketAsyncEventArgs e)+WaitForConnectWithCancellation(?)
   at async ValueTask<Stream> System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(string host, int port, HttpRequestMessage initialRequest, bool async, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at async ValueTask<Stream> System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(string host, int port, HttpRequestMessage initialRequest, bool async, CancellationToken cancellationToken)
   at async ValueTask<ValueTuple<Stream, TransportContext>> System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, bool async, CancellationToken cancellationToken)
   at async ValueTask<HttpConnection> System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, bool async, CancellationToken cancellationToken)
   at async Task System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(QueueItem queueItem)
   at async ValueTask<T> System.Threading.Tasks.TaskCompletionSourceWithCancellation<T>.WaitWithCancellationAsync(CancellationToken cancellationToken)
   at async ValueTask<T> System.Net.Http.HttpConnectionPool+HttpConnectionWaiter<T>.WaitForConnectionAsync(bool async, CancellationToken requestCancellationToken)
   at async ValueTask<HttpResponseMessage> System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, bool async, bool doRequestAuth, CancellationToken cancellationToken)
   at async ValueTask<HttpResponseMessage> System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, bool async, CancellationToken cancellationToken)
   at async Task<Stream> System.Net.Http.HttpClient.GetStreamAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at async Task<Stream> Flow.Launcher.Infrastructure.Http.Http.GetStreamAsync(Uri url, CancellationToken token) in C:/projects/flow-launcher/Flow.Launcher.Infrastructure/Http/Http.cs:line 162
   at async Task<UpdateManager> Flow.Launcher.Core.Updater.GitHubUpdateManagerAsync(string repository) in C:/projects/flow-launcher/Flow.Launcher.Core/Updater.cs:line 127
   at async Task Flow.Launcher.Core.Updater.UpdateAppAsync(IPublicAPI api, bool silentUpdate) in C:/projects/flow-launcher/Flow.Launcher.Core/Updater.cs:line 43

